deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,35 @@
+{
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
+      'direct_dependent_settings': {
+        'include_dirs': ['.'],
+      },
+      'sources': ['simdutf.cpp'],
+      'cflags_cc': [ '-std=gnu++11', '-Wno-unused-command-line-argument' ],
+      'conditions': [
+        ['target_arch in "x64" and OS!="win"', {
+          'variables': {
+            'supports_vbmi2%': '<!(<!(echo ${CC_target:-${CC:-cc}}) -MM -E simdutf.cpp -mavx512vl -mavx512vbmi2 2>/dev/null || true)'
+          },
+          'conditions': [
+            ['supports_vbmi2!=""', {
+              'cflags': [
+                '-mavx512f',
+                '-mavx512vbmi2',
```

Comment:
Do these flags need to be set at the compiler level? That seems wrong for SIMD source which auto-detects processor features, and it comes with the risk of the compiler emitting AVX512 instructions in code that isn’t supposed to be restricted to AVX512 processors, right?

Reply:
These flags are required for our CI builds since most of them have GCC-8, which depends on these flags.

Reply:
But what addresses the risk of emitting invalid instructions? I can see that this fails locally for me with gcc-8, but that seems like a bug in the intrinsics headers that ship with GCC 8. If we want or need this to work with GCC 8, then we should probably work around this in a way that’s similar to what GCC does in those headers, which is to temporarily set change the compiler target via `#pragma GCC target(...)` around the problematic `#include`(or `#include`s?) in simdutf.cpp.

Reply:
I updated the pull request with the changes of simdutf 2.0.7